### ISSUE TITLE
WatchOS 2.0: Library not loaded

### ISSUE THREAD

**Issue by RZAtwood** [CocoaPods/CocoaPods#4180]
I can't get CocoaPods working w/ a WatchKit 2 extension. CocoaPods 0.38 added watchOS support, as did Alamofire 2.0. But I see the following in my WatchKit app when I try to use CocoaPods-installed Alamofire.

```
dyld: Library not loaded: @rpath/Alamofire.framework/Alamofire
Referenced from: ...
Reason: image not found
```

Steps to Reproduce:

1) Create new project (TestApp) using "iOS App with WatchKit App" template.
2) Add Podfile (below).

```
inhibit_all_warnings!
use_frameworks!

target 'TestApp' do
	platform :ios, '9.0'
	pod 'Alamofire', '~> 2.0'
end


target 'TestApp WatchKit Extension' do
	platform :watchos, '2.0'
	pod 'Alamofire', '~> 2.0'
end
```

3) Run pod install.
4) Import Alamofire in ExtensionDelegate.swift.
5) Build then run the WatchKit App.
6) Get error.

**Comment by segiddins** [comment]
@neonichu isn't this only fixed in the beta, since at first we didn't embed frameworks in watchos2 apps?

**Comment by neonichu** [comment]
Yep, this has been solved in #3920 which is part of the 0.39 beta.

**Comment by RZAtwood** [comment]
Thanks! Confirmed everything works in 0.39.0.beta.4.
